Formula/cascadia.rb

```diff
@@ -0,0 +1,21 @@
+class Cascadia < Formula
+  desc "Go cascadia package command-line CSS selector"
+  homepage "https://github.com/suntong/cascadia"
+  url "https://github.com/suntong/cascadia/archive/refs/tags/v1.2.6.tar.gz"
+  sha256 "eb7f88f23f176e74d1aeb3b34efeb7763de8ad5445566bc92ed9fee33e9cebd1"
+  license "MIT"
+  head "https://github.com/suntong/cascadia.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    # go mod init/tidy are needed until https://github.com/suntong/cascadia/issues/13 is fixed
+    system "go", "mod", "init", "github.com/suntong/cascadia"
+    system "go", "mod", "tidy"
```

Comment:
maybe wait until the new release coming out?

Reply:
If there is a new release coming out. Author initially seemed reluctant:

> Yes, `GO111MODULE=off` is the way I do the build. 
> 
> I'll hold off turning it on as long as I could, as I personally believe it'll bring more mess to my simple project than doing it good. Huge projects with tons of dependencies is another different story.
> 
> _Originally posted by @suntong in https://github.com/suntong/cascadia/issues/12#issuecomment-1374699422_

Though this message seemed to imply that they might be willing to figure out/fix the blocker, in which case it might land:

> Please take a look at 
> https://github.com/suntong/cascadia/actions/runs/1596372916/jobs/2135287222
> 
> I.e., without `go.mod` it builds just fine. 
> 
> I'll take a look at the CI/CD build again tomorrow.
> 
> _Originally posted by @suntong in https://github.com/suntong/cascadia/issues/13#issuecomment-1374697805_

Reply:
Without a go mod/lock file it's violating https://docs.brew.sh/Acceptable-Formulae#we-dont-like-install-scripts-that-download-unversioned-things

Reply:
Alright then. go mod file will be there in the next version.

Reply:
> Looks like the changes from #14 will land in `v1.2.7`:
> 
> - https://github.com/suntong/cascadia/releases/tag/v1.2.7
> 
> _Originally posted by @0xdevalias in https://github.com/suntong/cascadia/issues/12#issuecomment-1374961166_

---

Bumped to `1.2.7` + removed the `go.mod` hack in [`b6e643b` (#120044)](https://github.com/Homebrew/homebrew-core/pull/120044/commits/b6e643b3d97f455d27a04f65c456563ce85035f0)